### ISSUE TITLE
Merging contacts cause synchronization problem

### ISSUE THREAD

**Issue by samuelsov** [agileware/au.com.agileware.civiquickbooks#30]
To reproduce:
1. create 2 contacts with different names (contact1=OrgName and contact2=OrgName inc)
2. let the `Contact push` task create both contact in Quickbooks
3. merge the 2 contacts in CiviCRM, keep contact1 but rename the remaining contact as contact2

This gives an error `Matches QBO Contact %1, which is already synced to %2 (%3). Deduplication is required.`

What is the best way to fix this from a user point of view ? Should the user delete the client in QBO ?

**Comment by agileware-justin** [comment]
@samuelsov yes, the same action should be performed in QB - in this case, delete contact 2.
